### ISSUE TITLE
text label for X close buttons

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#525]
The ModalBox and Popover examples should be updated to use "Close" for the text string on the X close button according to https://github.com/patternfly/patternfly-next/pull/442#discussion_r207653094
